desktop/components/current_auctions/README.md

```diff
@@ -0,0 +1,58 @@
+# Current Auctions
+
+A React component that displays a bar of four current Auctions.
+
+## Usage
+
+If using within a React context:
+
+```javascript
+import { CurrentAuctions } from 'components/current_auctions'
+
+class Page extends Component {
+  render() {
+    return (
+      <CurrentAuctions
+        sales={sales}
+      />
+    )
+  }
+}
+```
+
+If using within a non-React context, `index.jsx` will automatically mount provided there is a `react-mount-current-auctions` ID somewhere present in the DOM:
```

Comment:
Maybe better here to allow you to pass in the id of the target DOM element that you want it to mount to.